vips/vip-440/bsctestnet.ts

```diff
@@ -0,0 +1,256 @@
+        params: [POOL_REGISTRY, 0],
+        dstChainId: LzChainId.sepolia,
+      },
+      {
+        target: vsUSDS,
+        signature: "setProtocolSeizeShare(uint256)",
+        params: [parseUnits("0.05", 18)],
+        dstChainId: LzChainId.sepolia,
+      },
+
+      {
+        target: COMPTROLLER,
+        signature: "setActionsPaused(address[],uint8[],bool)",
+        params: [[vsUSDS], [2], true],
+        dstChainId: LzChainId.sepolia,
+      },
+      // set converters
+      {
+        target: USDT_PRIME_CONVERTER,
+        signature: "setConversionConfigs(address,address[],(uint256,uint8)[])",
+        params: [BaseAssets[0], [USDS], [[CONVERSION_INCENTIVE, 1]]],
+        dstChainId: LzChainId.sepolia,
+      },
+      {
+        target: USDC_PRIME_CONVERTER,
+        signature: "setConversionConfigs(address,address[],(uint256,uint8)[])",
+        params: [BaseAssets[1], [USDS], [[CONVERSION_INCENTIVE, 1]]],
+        dstChainId: LzChainId.sepolia,
+      },
+      {
+        target: WBTC_PRIME_CONVERTER,
+        signature: "setConversionConfigs(address,address[],(uint256,uint8)[])",
+        params: [BaseAssets[2], [USDS], [[CONVERSION_INCENTIVE, 1]]],
+        dstChainId: LzChainId.sepolia,
+      },
+      {
+        target: WETH_PRIME_CONVERTER,
+        signature: "setConversionConfigs(address,address[],(uint256,uint8)[])",
+        params: [BaseAssets[3], [USDS], [[CONVERSION_INCENTIVE, 1]]],
+        dstChainId: LzChainId.sepolia,
+      },
+      {
+        target: XVS_VAULT_CONVERTER,
+        signature: "setConversionConfigs(address,address[],(uint256,uint8)[])",
+        params: [BaseAssets[4], [USDS], [[CONVERSION_INCENTIVE, 1]]],
+        dstChainId: LzChainId.sepolia,
+      },
+
+      {
+        target: USDT_PRIME_CONVERTER,
+        signature: "setConversionConfigs(address,address[],(uint256,uint8)[])",
+        params: [BaseAssets[0], [sUSDS], [[CONVERSION_INCENTIVE, 1]]],
+        dstChainId: LzChainId.sepolia,
+      },
+      {
+        target: USDC_PRIME_CONVERTER,
+        signature: "setConversionConfigs(address,address[],(uint256,uint8)[])",
+        params: [BaseAssets[1], [sUSDS], [[CONVERSION_INCENTIVE, 1]]],
+        dstChainId: LzChainId.sepolia,
+      },
+      {
+        target: WBTC_PRIME_CONVERTER,
+        signature: "setConversionConfigs(address,address[],(uint256,uint8)[])",
+        params: [BaseAssets[2], [sUSDS], [[CONVERSION_INCENTIVE, 1]]],
+        dstChainId: LzChainId.sepolia,
+      },
+      {
+        target: WETH_PRIME_CONVERTER,
+        signature: "setConversionConfigs(address,address[],(uint256,uint8)[])",
+        params: [BaseAssets[3], [sUSDS], [[CONVERSION_INCENTIVE, 1]]],
+        dstChainId: LzChainId.sepolia,
+      },
+      {
+        target: XVS_VAULT_CONVERTER,
+        signature: "setConversionConfigs(address,address[],(uint256,uint8)[])",
+        params: [BaseAssets[4], [sUSDS], [[CONVERSION_INCENTIVE, 1]]],
+        dstChainId: LzChainId.sepolia,
+      },
```

Comment:
AFAIK, we could "merge" these commands with the previous ones, configuring the conversions on each converter with one command for both assets (USDS and sUSDS). For example, for the USDT converter:

```
      {
        target: USDT_PRIME_CONVERTER,
        signature: "setConversionConfigs(address,address[],(uint256,uint8)[])",
        params: [BaseAssets[0], [USDS, sUSDS], [[CONVERSION_INCENTIVE, 1], [CONVERSION_INCENTIVE, 1]]],
        dstChainId: LzChainId.sepolia,
      },
```